common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -66,16 +66,6 @@ trait ABTestSwitches {
     exposeClientSide = true
   )
 
-  Switch(
-    ABTests,
-    "ab-commercial-ad-verification-q-a",
-    "Test the impact of verifiyng ads for QA before rolling it out to wider audiance",
-    owners = Seq(Owner.withGithub("guardiancosta")),
-    safeState = Off,
-    sellByDate = new LocalDate(2019, 9, 30),
-    exposeClientSide = true
-  )
-
```

Comment:
Do we not need a `Switch` for `ab-commercial-ad-verification` to replace the one your removing?

Reply:
Ag ok I see a `Switch` for `ab-commercial-ad-verification` already exists in the file